<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinePrintWidget</class>
 <widget class="QWidget" name="LinePrintWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_numSets">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>112</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Number of Sets</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numSets">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_StartXY">
       <property name="text">
        <string>Start X, Y (mm)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="startX">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="value">
        <double>15.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="startY">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_setSpacing">
       <property name="text">
        <string>Set Spacing (mm)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="setSpacing">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Set 1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Set 2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Set 3</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Number of Lines</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Line Spacing (mm)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Line Length (mm)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Droplet Spacing (um)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Jetting Frequency (Hz)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Printing Velocity (mm/s)</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>1000</string>
      </property>
     </item>
     <item row="0" column="5">
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="8,5,1">
     <item>
      <widget class="QTextEdit" name="consoleOutput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="SvgView" name="SVGViewer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="interactive">
          <bool>false</bool>
         </property>
         <property name="transformationAnchor">
          <enum>QGraphicsView::AnchorViewCenter</enum>
         </property>
         <property name="resizeAnchor">
          <enum>QGraphicsView::AnchorViewCenter</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="viewArrows">
           <property name="text">
            <string>View Arrows</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="viewMovePaths">
           <property name="text">
            <string>View Move Paths</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="printPercentSlider">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
       <property name="sliderPosition">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="clearConsole">
       <property name="text">
        <string>Clear Console</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveConfig">
       <property name="text">
        <string>Save Configuration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startPrint">
       <property name="text">
        <string>Start Print</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SvgView</class>
   <extends>QGraphicsView</extends>
   <header>svgview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
